content/tutorial/02-advanced-orders/03-pre-signed-orders/01-create-pre-signed-order/README.md

```diff
@@ -0,0 +1,370 @@
+
+#### `Safe` address
+
+The `Safe` address is the address of the smart contract wallet that we're going to be using to make the trade. Replace the `safeAddress` with the address of your `Safe` wallet:
+
+```typescript
+/// file: run.ts
+// ...
+
+export async function run(provider: Web3Provider): Promise<unknown> {
+  // ...
+  const safeAddress = '0x075E706842751c28aAFCc326c8E7a26777fe3Cc2'
+  // ...
+}
+```
+
+### Contract instance
+
+Now that we have the contract address and the ABI, we can create the contract instance. As we are just going to be ABI encoding the `setPreSignature` function, we don't need to connect to the contract with a `signer` or a `provider`:
+
+```typescript
+/// file: run.ts
+// ...
+
+export async function run(provider: Web3Provider): Promise<unknown> {
+  // ...
+  const settlementContract = new Contract(COW_PROTOCOL_SETTLEMENT_CONTRACT_ADDRESS[chainId], abi)
+  // ...
+}
+```
+
+### Connect to the Safe
+
+Now that we have the `Safe` address and associated helper functions, we can connect to the `Safe`:
+
+```typescript
+/// file: run.ts
+// ...
+
+export async function run(provider: Web3Provider): Promise<unknown> {
+  // ...
+  const { safeApiKit, safeSdk } = await getSafeSdkAndKit(safeAddress)
+  // ...
+}
+```
+
+### Get a quote
+
+As per normal, we are going to request a quote to buy COW tokens with wxDAI using `OrderBookApi`:
+
+```typescript
+/// file: run.ts
+// ...
+import {
+  SupportedChainId,
+  OrderBookApi,
++++  SigningScheme,+++
++++  OrderQuoteRequest,+++
++++  OrderQuoteSideKindSell,+++
++++  OrderCreation,+++
+  COW_PROTOCOL_SETTLEMENT_CONTRACT_ADDRESS
+} from '@cowprotocol/cow-sdk'
+// ...
+
+export function run(provider: Web3Provider): Promise<unknown> {
+  // ...
+  const sellAmount = '1000000000000000000';
+  const sellToken = '0xe91D153E0b41518A2Ce8Dd3D7944Fa863463a97d';
+  const buyToken = '0x177127622c4A00F3d409B75571e12cB3c8973d3c';
+
+  const quoteRequest: OrderQuoteRequest = {
+    sellToken,
+    buyToken,
+    receiver: safeAddress,
+    sellAmountBeforeFee: sellAmount,
+    kind: OrderQuoteSideKindSell.SELL,
+    appData: appDataContent,
+    appDataHash: appDataHex,
+    from: ownerAddress,
+  }
+
+  const { quote } = await orderBookApi.getQuote(quoteRequest);
+  // ...
+}
+```
+
+### Submit the order
+
+Now that we have the quote, we can submit the order to the order book. As we're using `PreSign`, we need to pay special attention to the fields: 
+
+- `from`: the address of the `Safe` wallet
+- `signature`: empty bytes, i.e. `0x`
+- `signingScheme`: `SigningScheme.PRESIGN`
+
+```typescript
+/// file: run.ts
+// ...
++++import { BigNumber, Contract, ethers } from 'ethers'+++
+// ...
+
+export function run(provider: Web3Provider): Promise<unknown> {
+  // ...
+  const order: OrderCreation = {
+    ...quote,
+    sellAmount,
+    buyAmount: BigNumber.from(quote.buyAmount).mul(9950).div(10000).toString(),
```

Comment:
Took me awhile to figure out but you are applying the slippage here!
We do need to work on adding this kind of thing to the SDK.